lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -123,7 +124,7 @@ export default function FilePickerApp({ onSubmit }) {
         <div className="FilePickerApp__left-col">Assignment content</div>
         <div className="FilePickerApp__right-col">
           {content ? (
-            <i>{contentDescription(content)}</i>
+            <i data-testid="content-summary">{contentDescription(content)}</i>
```

Comment:
Could we use `em` here instead? `i` is a formatting element, while `em` is more semantic. Both should have identical default browser styling.

Reply:
Apparently in the living HTML spec the `<i>` tag [has been retconned](https://html.spec.whatwg.org/#the-i-element) to be semantic but in a different way than `em`. In this case it isn't clear to me which is really appropriate - _em_ means _emphasis_ whereas _i_ means "in an alternate voice or mood or of a different quality". I'm going to punt on this change for now and we can revisit later.